### PR TITLE
update gemm interface to support batchmatmul

### PR DESCRIPTION
**Pull request by minminsun**

### CODE REVIEW

File: tao_compiler/mlir/xla/ral/context/stream_executor_based_impl.cc [hunk 1]
```diff
@@ -280,17 +296,18 @@ void ral_gemm(ExecutionContext* ctx, void* stream_handle, MemRefType<InT, 2> A,
   }
 
 #if defined(PLATFORM_ALIBABA) and defined(ENABLE_BLADE_GEMM)
-  if (std::is_same<InT, Eigen::half>::value ||
-      std::is_same<InT, float>::value) {
+  {
```
Comment (lfengad) [inline tao_compiler/mlir/xla/ral/context/stream_executor_based_impl.cc:299]: 这里需不需要用环境变量确认BladeGemm的开启关闭？

**Review by lfengad — COMMENTED** [review]
LGTM

File: tao_compiler/mlir/xla/ral/context/stream_executor_based_impl.cc (hunk 1 above)
Comment (minminsun) [reply tao_compiler/mlir/xla/ral/context/stream_executor_based_impl.cc:299]: Yes, and this is added inside the gemm() function.

**Comment by lfengad** [comment]
LGTM

**Review by qiuxiafei — APPROVED** [review]

**Pull request merged**